ccd_tempreture taking astropy unit (#150)
sunpy/sunkit-instruments: changelog/150.feature.3.rst

Reviewer: Why is there .3 in the filename?

Reviewer: Delete this file.

> ### Added
> - Support for `ccd_temperature` as an Astropy Quantity in the `get_response` function.
> ### Changed
> - Updated `get_response` to enforce `ccd_temperature` as an Astropy Quantity with `@u.quantity_input` decorator.

Reviewer: ```suggestion
Updated `~.get_response` to enforce ``ccd_temperature`` as an `astropy.units.Quantity`
```
Not sure if this will pass the doc build.

### Added
- Support for `ccd_temperature` as an Astropy Quantity in the `get_response` function.
### Changed
- Updated the `get_response` function signature to accept `ccd_temperature` as an Astropy Quantity.
- Modified the function to handle `ccd_temperature` inputs with and without units, converting them to degrees Celsius.

> - Updated the docstring of the `get_response` function to reflect the new parameter type and units requirement.

Reviewer: This changelog is too verbose.
